src/theme/components/button.ts

```diff
@@ -37,6 +37,7 @@ export const Button = defineStyleConfig({
     }),
 
     primaryPortableError: defineStyle({
+      height: '48px',
```

Comment:
it is better not hardcode the height, but if just for quick thats fine

Reply:
Yes I just followed the previous style used for Primary. I agree, we need to revisit the styles